src/main/java/green/projectile/Projectile.java

```diff
@@ -46,8 +54,15 @@ private double getVerticalVelocity() {
     }
 
     //method to get time at which the projectile reaches the x intercept
-    private double getTimeAtXInt() {
+    public double getTotalTime() {
         double vertical = getVerticalVelocity();
         return (vertical + Math.sqrt(vertical * vertical - 2 * GRAVITY * getY())) / GRAVITY;
     }
+
+    //this method returns the same number as getTotalTime, but projectileGraph only works properly when I use this one. Will probably
+    //just delete the other but I'd like to figure out why first
```

Comment:
getTotalTime() calls getY() which will change based on the "seconds". So they won't always return the same number.